api/tests/core/users/test_api.py

```diff
@@ -336,116 +336,53 @@ def test_bulk_unsuspend_account(self):
-        expiration_date = datetime.now() + timedelta(hours=1)
-        missing_current_email_token_payload = dict(new_email="newemail@mail.com")
-        missing_current_email_token = encode_jwt_payload(missing_current_email_token_payload, expiration_date)
-
-        missing_new_email_token_payload = dict(current_email="oldemail@mail.com")
-        missing_new_email_token = encode_jwt_payload(missing_new_email_token_payload, expiration_date)
-
-        missing_exp_token_payload = dict(new_email="newemail@mail.com")
-        missing_exp_token = encode_jwt_payload(missing_exp_token_payload)
-
-        # When
-        with pytest.raises(jwt.exceptions.InvalidTokenError):
-            users_api.change_user_email(missing_current_email_token)
-            users_api.change_user_email(missing_new_email_token)
-            users_api.change_user_email(missing_exp_token)
+        assert old_user is None
 
-        # Then
-        old_user = User.query.filter_by(email="oldemail@mail.com").first()
-        assert old_user is not None
-        new_user = User.query.filter_by(email="newemail@mail.com").first()
-        assert new_user is None
+        assert UserSession.query.filter_by(userId=user.id).first() is None
 
-    @freeze_time("2020-10-15 09:00:00")
     def test_change_user_email_new_email_already_existing(self):
         # Given
-        users_factories.UserFactory(email="newemail@mail.com", firstName="UniqueNameForEmailChangeTest")
-        expiration_date = datetime.now() + timedelta(hours=1)
-        token_payload = dict(current_email="oldemail@mail.com", new_email="newemail@mail.com")
-        token = encode_jwt_payload(token_payload, expiration_date)
+        user = users_factories.UserFactory(email="oldemail@mail.com", firstName="UniqueNameForEmailChangeTest")
+        other_user = users_factories.UserFactory(email="newemail@mail.com")
 
         # When
-        users_api.change_user_email(token)
+        with pytest.raises(users_exceptions.EmailExistsError):
+            users_api.change_user_email(user.email, other_user.email)
 
         # Then
-        old_user = User.query.filter_by(email="oldemail@mail.com").first()
-        assert old_user is None
-        new_user = User.query.filter_by(email="newemail@mail.com").first()
-        assert new_user is not None
+        user = User.query.get(user.id)
+        assert user.email == "oldemail@mail.com"
 
-    @freeze_time("2020-10-15 09:00:00")
-    def test_change_user_email_current_email_not_existing_anymore(self):
-        # Given
-        expiration_date = datetime.now() + timedelta(hours=1)
-        token_payload = dict(current_email="oldemail@mail.com", new_email="newemail@mail.com")
-        token = encode_jwt_payload(token_payload, expiration_date)
+        other_user = User.query.get(other_user.id)
+        assert other_user.email == "newemail@mail.com"
 
+    def test_change_user_email_current_email_not_existing_anymore(self):
```

Comment:
Pourquoi "anymore" ?
```suggestion
    def test_change_email_user_not_existing(self):
```